dockerhub/travis-ci-image.dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apt-get update -y && apt-get install -qy \
         librados-dev \
         liberasurecode-dev \
         ceph \
-    && apt-get clean -y
```

Comment:
You did not read my commit message :(

https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#minimize-the-number-of-layers

> Note: The official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.

https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105